requirements/base.txt

```diff
@@ -17,3 +17,4 @@ edx-ccx-keys==0.2.1
 edx-django-release-util==0.3.0
 edx-opaque-keys==0.4.0
 edx-rest-api-client==1.4.0          # Apache 2.0
+edx-drf-extensions==1.2.4
```

Comment:
does edx-enterprise-data need to be pulled in here too? (i'm guessing it's not on pypi yet...)